src/backend/joanie/core/flows/order.py

```diff
@@ -228,6 +200,40 @@ def failed_payment(self):
         Mark order instance as "failed_payment".
         """
 
+    def init(self, billing_address=None):
```

Comment:
I don't understand this change. The `assign` function is empty, why doing this ?

Reply:
Because, assign is a state transition, which does nothing more, like many others.
I think it's clearer like that.

Reply:
So `init` should be moved into Order model nope ?

Reply:
Yes, maybe

Reply:
I'll do this in the next PR which removes all flow.init calls from the tests.